pkg/parser/parser.go

```diff
@@ -154,6 +197,138 @@ func (p *Parser) ParseFromFile(path string) (*Result, error) {
 	}
 
 	// Parse again with the file contents supplied
-	p.FilesContents = filesContents
+	p.filesContents = filesContents
 	return p.Parse(string(config))
 }
+
+func (p *Parser) ContentHash() string {
+	digest := sha256.New()
+
+	for key, value := range p.filesContents {
```

Comment:
This doesn't look right. Content hash is per file with docker arguments and not for all Dockerfiles.

Reply:
Good catch, fixed in https://github.com/cirruslabs/cirrus-cli/pull/58/commits/6302f7db06b8cefddafe2c7e7260253bf9d6a70a.